lib/js/components/SelectionTile/SelectionTile.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+	<div
+		class="selectionTile"
+		:class="[
+			{
+				'-selected': isSelected,
+				'-default': state === SELECTION_TILE_STATE.DEFAULT,
+				'-disabled': state === SELECTION_TILE_STATE.DISABLED,
+				'-loading': state === SELECTION_TILE_STATE.LOADING,
+			},
+		]"
+		@click="updateIsSelected(!isSelected)"
+	>
+		<div class="selectionTile__wrapper">
+			<radio-button
+				v-if="type === SELECTION_TILE_TYPE.RADIO_BUTTON"
+				:size="RADIO_BUTTON_SIZE.X_SMALL"
+				:is-selected="isSelected"
+				:state="SELECTION_CONTROL_STATE_MAP[state]"
+				@update:isSelected="updateIsSelected($event)"
+			/>
+			<checkbox
```

Comment:
Radio and Checkbox components has the same API right now. We could use `<component is="">` to guarantee noone forgets to update 2nd component when introducing changes in the first one